pallets/pool-system/src/impls.rs

```diff
@@ -553,8 +553,8 @@ mod benchmarks_utils {
 			Pallet::<T>::close_epoch(RawOrigin::Signed(admin.clone()).into(), pool_id).unwrap();
 		}
 
-		fn benchmark_give_ausd(account: &T::AccountId, balance: T::Balance) {
-			T::Tokens::mint_into(AUSD_CURRENCY_ID, account, balance).unwrap();
+		fn bench_mint_pool_currency_into(account: &T::AccountId, balance: T::Balance) {
```

Comment:
Thinking on the name of this function, WDYM: `bench_pool_user()`?.

I say that because what the function does should not be relevant to the user of it. As a last resort, the API user would want to configure a *pool user*. No matter what it implies. That way if tomorrow, it implies more things than "minting" we do not need to change the name.

Reply:
I partially agree. The question is whether the definition of _a pool user_ is clear. To me, it is an account with sufficient funds to invest and the `Investor` permission. What does it mean to you? Maybe we want to have finer granularity and have one call to fund an account and separate calls for preparing certain roles.

Reply:
For now, I went with `bench_investor_setup`. Depending on your response, I might introduce finer granularity.

Reply:
Not sure what direction it will grow, TBH, but I like a lot the new name `bench_investor_setup()`, more than `bench_pool_user` 👍🏻